lib/danger/ci_source/circle.rb

```diff
@@ -6,7 +6,7 @@ module CISource
     class CircleCI < CI
       def self.validates?(env)
         return !env["CIRCLE_BUILD_NUM"].nil? &&
-               !env["CI_PULL_REQUEST"].nil?
+          !env["CI_PULL_REQUEST"].nil?
```

Comment:
Align the operands of a condition in a `return` statement spanning multiple lines.